app/html/petition.html

```diff
@@ -99,17 +99,63 @@
 			</div>
 
 		</div>
-
+		<!--Start of Petition Form-->
+		<div class="container-fluid">
+		<div class="row">
+		<div class="col-md-offset-3 col-md-6">
+		<form id="petitionform">
+			<h3 class="darkgrey" style="color:darkgrey;">Sign the Petition!</h3>
+			<div class="petitionrequired" style="color:darkgrey;">* Required</div>
+				<br>
```

Comment:
Instead of using branch statements I recommend setting the bottom margin of each row. This makes it display better on mobile.

```
<div class="row petition-row">
...
</div>

.petition-row {
  margin-bottom: <whatever value here>px;
}
```